src/test/kotlin/org/rust/ide/formatter/RsFormatterTest.kt

```diff
@@ -90,6 +90,16 @@ class RsFormatterTest : RsFormatterTestBase() {
         """)
     }
 
+    fun `test indent where clause on`() {
+        custom().INDENT_WHERE_CLAUSE = true
+        doTest()
```

Comment:
Let's inline test code from files
It significantly simplifies reading
See `test align where bounds off` test as example